packages/amplify-e2e-core/src/categories/auth.ts

```diff
@@ -1579,3 +1579,58 @@ export function updateAuthWithoutTrigger(cwd: string, settings: any): Promise<vo
       });
   });
 }
+
+export function updateAuthWithGroupTrigger(cwd: string, settings: any): Promise<void> {
+  return new Promise((resolve, reject) => {
+    spawn(getCLIPath(), ['update', 'auth'], { cwd, stripColors: true })
+      .wait('What do you want to do?')
+      .send(KEY_DOWN_ARROW)
```

Comment:
we have a dedicated method for this with repetitions support, also this should be a `SingleSelect`,  not arrow key usage for better control.